pycbc/events/significance.py

```diff
@@ -0,0 +1,253 @@
+# Copyright (C) 2022 Gareth Cabourn Davies
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the
+# Free Software Foundation; either version 3 of the License, or (at your
+# option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+#
+# =============================================================================
+#
+#                                   Preamble
+#
+# =============================================================================
+#
+
+"""
+This module contains functions to calculate the significance
+through different estimation methods of the background, and functions that
+read in the associated options to do so.
+"""
+import logging
+import copy
+import numpy as np
+from pycbc.events import coinc
+from pycbc.events import trigger_fits as trstats
+
+
+def n_louder(bstat, fstat, dec, **kwargs):  # pylint:disable=unused-argument
```

Comment:
I'm not following why this function is wanted, ie why we need to define a function that does nothing but call an existing function from elsewhere with the same arguments. 
(Also if we are doing this, I think the **kwargs should be passed through to the coinc function call ..)

Reply:
The reason for this is to have the n_louder calculations for significance in one place

If apply_rerank isn't used anymore, I will move the code out of the coincidence module. I'll pass the kwargs as appropriate

Reply:
I'm not sure what 'apply_rerank' is here.

Reply:
https://github.com/gwastro/pycbc/blob/master/bin/all_sky_search/pycbc_apply_rerank - which is not used in the example search, and I'm not sure on its usage or history

Reply:
Anyway I don't mind if you move the actual coinc n louder function to this module, I just don't want content-free function definitions that make the code as a whole harder to understand and debug - functions with similar or identical names in different modules are not great on this score

Reply:
Ah, in any case you could change the statement in pycbc_apply_rerank to import from significance ?

Reply:
Update it now, unfortunately I don't know how to test it

Reply:
I wouldn't worry, given that it is not used in the workflow atm